languages/go/bitwarden_client.go

```diff
@@ -45,8 +45,8 @@ func NewBitwardenClient(apiURL *string, identityURL *string) (*BitwardenClient,
 	}, nil
 }
 
-func (c *BitwardenClient) AccessTokenLogin(accessToken string) error {
-	req := AccessTokenLoginRequest{AccessToken: accessToken}
+func (c *BitwardenClient) AccessTokenLogin(accessToken string, statePath *string) error {
```

Comment:
Nit: We're calling this new variable here `statePath`. In Python we've called it `state_file_path`. In Rust it's just `state_file`.

Should we decide on a consistent naming for them all?

Reply:
Yes, that's a great point. I think `statePath` is fairly clear, any thoughts on that?

I have a draft PR here for the other places in the SDK & in the Python wrapper: https://github.com/bitwarden/sdk/pull/587

Reply:
Yeah, that works for me 👍